extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -288,7 +288,7 @@ export function getCompletionItemsFromSpecs(specs: Fig.Spec[], terminalContext:
 		}
 	}
 
-	if (!specificSuggestionsProvided) {
+	if (!specificSuggestionsProvided && (filesRequested === foldersRequested)) {
```

Comment:
This is a little hard to read, does it actually mean if neither files or folders requested OR both files and folders requests?

Reply:
It means don't provide generic commands when just one of them is provided - because that means it was requested by an arg of cd for example